### ISSUE TITLE
Policy for updates/changes

### ISSUE THREAD

**Issue by xrotwang** [clld/concepticon-data#108]
Just came across a concept list that has been renamed from release 0.1 to 1.0, `Shiro-1973-200` is now `Hattori-1973-200`. We should come up with a policy document describing how these things are handled. Alternatives may be
- Just do it, i.e. changes will have to be discovered by digging through the git history.
- Don't do it, i.e. no identifiers can ever disappear. This may require some sort of obsolescence flag, to mark expired objects.
- Have a list of errata - possibly implemented as labeled GitHub issues - to make relevant changes more transparent.

I think I'm leaning towards the third option, hoping that changes won't have to happen often.

**Comment by LinguList** [comment]
I agree. The problem with Shiro was: I figured that this was not the
family name but the first name, so Hattori is the family name, but they
would write Shiro Hattori, and I confused it with Chinese spelling, like
Mao Zedong, where the family name precedes the given name. There are
some similar things with concepts, I guess, where I had to change
identifiers, since it turned out they were wrong (but usually not
occuring in any specific list).

Having an "errata-policy" seems to be easier to handle than keeping all
identifiers. I hope also, that this won't happen too often in the
future, but one never knows...
